core/services/pg/lease_lock_test.go

```diff
@@ -47,7 +48,7 @@ func Test_LeaseLock(t *testing.T) {
 
 		select {
 		case <-started2:
-		case <-time.After(5 * time.Second):
+		case <-time.After(cltest.WaitTimeout(t)):
```

Comment:
This is not related to this story, but this test is 100% flacky on my machine and @samsondav suggested this fix, which worked. I decided to include it into my ongoing work.